README.md

```diff
@@ -1,54 +1,30 @@
 [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![Build status](https://ci.appveyor.com/api/projects/status/77wjwjne4gwgou41/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/bhom-adapter/branch/master)
+[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/BHoM_Adapter/BHoM_Adapter.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=17&branchName=master)
+# Core BHoM_Adapter Repo
```

Comment:
For consistency with the BHoM ReadMe - can we either have a blank line above this header, or remove the blank line above the header in the BHoM ReadMe

Reply:
done similar to BHoM/Engine. Thanks